docs/tutorials/build-your-first-blazor-app/samples/2.x/ToDoSample/global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.300-preview2"
```

Comment:
Currently this should be `2.1.300-preview2-008523`, but were expecting an update to the preview 2 SDK soon.